extension/content_script.js

```diff
@@ -411,6 +411,23 @@ function adjustSpeed(speedDelta) {
 	timeTrackerView.updateSpeed(speed,sentence_id);
 }
 
+/* Change reading speed display. This has two types:
+	1. Update - Show WPM display when users change their speed setting
+	2. Toggle - Toggle WPM display when auto-mode is turned ON/OFF
+
+	Params: 
+	- type (str)
+*/
+function changeReadingSpeedDisplay(type) {
+	// TODO: Implement mutually exclusive animation strategy
+	let wpmDisplay = $("#speedContainer");
+	if (type === "update") {
+		if (!timer) { wpmDisplay.stop(true).fadeIn(250).delay(750).fadeOut(1000); };
```

Comment:
If not in auto mode, we only want this to show up when it is changed. -->
First, user clicks on it so it fades in, delays, then fades out. stop() does nothing, since hasn't started any animations. 
But, if user clicks on S/D again before the first animation cycle is complete, perhaps to rapidly change through WPM speeds, this will:
1. Stop current animation; we don't want it to fade any more
2. Clear the animation queue, so it doesn't get filled with a zillion cycles of fadeIn/fadeOut
3. Reset animation to full opacity; if it was partway fading out, this is what we want, if it was fadingIn/delaying it is unaffected
4. Continue on the origin path to fading out; wait 750ms, then fadeOut for 1s

Reply:
Nice! I was expecting a more complicated solution for this, but this one-liner seems to do it :D

Reply:
I tried so many things that were much more complicated before finding this :sob: :sob: :sob: (thanks jQuery for having such a good built-in animation queue system!)